src/security/src/main/kotlin/org/vaccineimpact/api/security/MontaguTokenAuthenticator.kt

```diff
@@ -3,13 +3,19 @@ package org.vaccineimpact.api.security
 import com.nimbusds.jwt.JWT
 import org.pac4j.core.credentials.TokenCredentials
 import org.pac4j.core.exception.CredentialsException
+import org.pac4j.core.profile.CommonProfile
 import org.pac4j.jwt.credentials.authenticator.JwtAuthenticator
 
 open class MontaguTokenAuthenticator(
         private val tokenHelper: WebTokenHelper,
         private val expectedType: TokenType
 ) : JwtAuthenticator(tokenHelper.signatureConfiguration)
 {
+    override fun validateToken(token: String?): CommonProfile?
```

Comment:
It'd be good to add a test for this, `canValidateDeflatedToken`?

Reply:
Currently this class is implicitly tested by `WebTokenHelperTests`. I think this is a reasonable approach to keep taking, in which case `can generate bearer token` tests this path.